config/moonguard.php

```diff
@@ -119,13 +119,6 @@
         /*
          * The notification channels that are used by default.
          */
-        'channels' => ['mail', 'slack'],
-
-        'slack' => [
-            /*
-             * The Slack webhook url setup.
-             */
-            'webhook_url' => env('SLACK_WEBHOOK_URL'),
-        ],
```

Comment:
It is not necessary to delete the `notifications.slack` configuration. Please restore it back.